<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SiemensTest</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>-1</y>
      <width>761</width>
      <height>441</height>
     </rect>
    </property>
    <property name="title">
     <string>SerialPort</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>COM:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cob">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>87</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Baudrate:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cob_baudrate">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>30</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnOpen">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>30</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnClose">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>30</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BtnConnect">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>100</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QPushButton" name="DisConnect">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>100</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>DisConnect</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>169</y>
       <width>731</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>LED1</string>
     </property>
     <widget class="QPushButton" name="btn1s">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>60</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Blink1s</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn500ms">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>60</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Blink500ms</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnOff">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Turn off</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn250ms">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>60</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Blink250ms</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtLED1State">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>30</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>30</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>LED1 State:</string>
      </property>
     </widget>
    </widget>
    <widget class="QTextBrowser" name="txtlog">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>731</width>
       <height>141</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>780</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
